src/i18n/en.json

```diff
@@ -511,7 +516,6 @@
     },
     "i18n-widget": {
         "country": "Country",
-        "language": "Language",
```

Comment:
Oh sorry, one last problem I missed:

This string also needs to stay here (in all languages). Otherwise, the heading in the "regular" I18nWidet is missing:

![image](https://user-images.githubusercontent.com/4048809/135750697-1fe4ce75-d622-4f60-a78f-e9f6379cc5b6.png)

Reply:
I'll just fix that myself, so we can finally get this PR merged. :D

Reply:
Oops, that's my bad. Thanks